determined-ai/determined
# fix: add non-scalar metric expectation to protobufs [DET-4893] [DET-4911]

## Description




## Test Plan

- run the provided noop experiment and check taht the api does not fail
- add master integ test. watch it fail, add a fix, watch it pass
![2021-01-29_15-12-27](https://user-images.githubusercontent.com/12127420/106336807-84c91180-6244-11eb-8a79-96ef16dcbcc4.png)





## Commentary (optional)

- for a more end-to-end test we need to seed the database for testing the api, mock the api response for testing the UI, or submit the test experiment and wait for it to finish before proceeding to have both. what do you suggest? 




## Checklist

- [x] remove the test commit
- [x] remove test scope limit
See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234
